okapi-core/src/main/java/org/folio/okapi/service/impl/ProcessModuleHandle.java

```diff
@@ -70,6 +70,21 @@ public ProcessModuleHandle(Vertx vertx, LaunchDescriptor desc, String id,
     }
   }
 
+  private Future<Void> waitPortOpen(NetClient c, int iter) {
+    return c.connect(port, "localhost")
+        .compose(socket ->
+                socket.close().compose(y -> {
+                  if (iter == 0) {
+                    return Future.failedFuture(
+                        messages.getMessage("11502", Integer.toString(port)));
+                  }
+                  Promise<Void> promise = Promise.promise();
+                  vertx.setTimer(100, x -> waitPortOpen(c, iter - 1)
+                      .onComplete(promise::handle));
+                  return promise.future();
+                }), x  -> Future.succeededFuture());
```

Comment:
`x` should be renamed to `noSocker` and vertically aligned with `socket` to better show that it is the second argument of the two-argument compose method.

Reply:
Thx, much better.